features/facebookImport/src/model/analysis.js

```diff
@@ -183,6 +183,18 @@ class AnalysisExecutionResult {
     get executionTime() {
         return this._executionTime;
     }
+
+    get reportJsonData() {
+        return {
+            analysisName: this.analysis.id,
+            activationStatus: this.analysis.active ? "ACTIVE" : "INACTIVE",
+            executionStatus: {
+                name: this.status.name,
+                message: this.status.message,
+            },
+            executionTime: this.executionTime.toFixed(2),
```

Comment:
Maybe toFixed(0) would be more adequate in this case. Do we need microseconds?

Reply:
Actually as we use `performance.now()` the time is already in microseconds, so we can use `toFixed(0)`